htdocs/lib/MapMarkers.js

```diff
@@ -254,10 +254,13 @@ FeatureMarker.prototype.draw = function() {
     div.style.color = this.color? this.color : '#000000';
     div.innerHTML   = this.symbol? this.symbol : '&#9679;';
 
-    var point = this.getProjection().fromLatLngToDivPixel(this.position);
-    if (point) {
-        div.style.left = point.x - this.symWidth/2 + 'px';
-        div.style.top = point.y - this.symHeight/2 + 'px';
+    // AF: this is of GMaps
```

Comment:
This getProjection() is GMaps specific and it's not needed for Leaflet.
You might want to move it to the GoogleMaps.js.
I've made a workaround for now.

Reply:
The fix here (in the FeatureMarker) is just to make things works.
The rendering of feature icons is not used by leaflet, since I choose to use the icons, for me it's more visible, but I can rework and use the symbols, like in GMaps. So we have consistency.